listener/types.go

```diff
@@ -216,6 +217,12 @@ func NewEthListenJob(jobType int, listener bridgeCore.Listener, subscriptionName
 	}
 }
 
+func (e *EthListenJob) Process() ([]byte, error) {
+	data, err := e.BaseJob.Process()
```

Comment:
Should we check the error != nil before sending it to stats?

Reply:
The sendErrToStats already checks if the error is nil so we don't need to check it everytime